src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5646,7 +5654,7 @@ CalcHeatBalanceInsideSurf( Optional_int_const ZoneToResimulate ) // if passed in
 				UpdateMoistureBalanceEMPD( SurfNum );
 				RhoVaporSurfIn( SurfNum ) = MoistEMPDNew( SurfNum );
 				//SUMC(ZoneNum) = SUMC(ZoneNum)-MoistEMPDFlux(SurfNum)*Surface(SurfNum)%Area
-
+				// TODO: Check with Jason that these equations were applied correctly. It's different than in his code.
```

Comment:
@jdwoods21 Your version had these equations as 

```FORTRAN
  !       SumHmAW(ZoneNum)= SumHmAW(ZoneNum)+HM(SurfNum)*Surface(SurfNum)%Area*  &
  !                         (Wwall(SurfNum)-ZoneAirHumRat(ZoneNum))
  !       SumHmARa(ZoneNum)= SumHmARa(ZoneNum)+HM(SurfNum)*Surface(SurfNum)%Area
  !       SumHmARaW(ZoneNum)= SumHmARaW(ZoneNum)+HM(SurfNum)*Surface(SurfNum)%Area*Wwall(SurfNum)
            SumHmAW(ZoneNum)= SumHmAW(ZoneNum) + FluxZone(SurfNum)*Surface(SurfNum)%Area !SumHmAW(ZoneNum)+HMassConvInFD(SurfNum)*Surface(SurfNum)%Area*  &
                    !       (RhoVaporSurfIn(SurfNum)-RhoVaporAirIn(SurfNum))

        RhoAirZone=PsyRhoAirFnPbTdbW(OutBaroPress,Mat(Surface(SurfNum)%Zone),PsyWFnTdbRhPb(Mat(Surface(SurfNum)%Zone),  &
              PsyRhFnTdbRhov(Mat(Surface(SurfNum)%Zone),RhoVaporAirIn(SurfNum),'RhoAirZone'),OutBaroPress))

         Wsurf=PsyWFnTdbRhPb(TempSurfInTmp(SurfNum), &
              PsyRhFnTdbRhov(TempSurfInTmp(SurfNum),RhoVaporSurfIn(SurfNum),'Wsurf'),OutBaroPress)

          SumHmARa(ZoneNum)= SumHmARa(ZoneNum)+HMshort(SurfNum)*Surface(SurfNum)%Area*RhoAirZone

         SumHmARaW(ZoneNum)= SumHmARaW(ZoneNum)+HMshort(SurfNum)*Surface(SurfNum)%Area*RVsurface(SurfNum)
```

Can you check and make sure they are applied correctly here?